megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -313,6 +314,11 @@ public CommonMenuBar() {
         fileUnitsReinforceRAT.addActionListener(this);
         fileUnitsReinforceRAT.setActionCommand(ClientGUI.FILE_UNITS_REINFORCE_RAT);
         submenu.add(fileUnitsReinforceRAT);
+        fileRefreshCache = new JMenuItem(Messages
```

Comment:
Can we please separate these? Might just be a personal preference, but I find it far easier to tell the differences between options when that's the case.

Reply:
I'm not going to rework the formatting of this file for the sake of a small insertion.

Reply:
I meant just for this piece, not the whole file.